src/hooks/useRegisterEmailForm.ts

```diff
@@ -0,0 +1,63 @@
+import {ChangeEvent, FormEvent, useState} from "react";
+import {useNavigate} from "react-router-dom";
+
+const createValidationInput = () => {
+    let newValidationInput = document.createElement('input');
+    newValidationInput.type = 'email';
+    newValidationInput.required = true;
+    return newValidationInput;
+}
+
+const useRegisterEmailForm = () => {
+    const [userEmail, setUserEmail] = useState<string>('');
+    let validationInput = createValidationInput();
+
+    const navigate = useNavigate();
+
+    const simpleEmailValidation = (email: string): boolean => {
+        return /\S+@\S+\.\S+/.test(email);
+    }
+
+    // email validation based on https://stackoverflow.com/a/13975255
+    const validateEmail = (): boolean => {
+        if (!userEmail) {
+            return false;
+        }
+
+        if(validationInput === null){
```

Comment:
```suggestion
        if (validationInput === null) {
```
It is advised to keep the code format across all your files, there are some cool shortcuts in Vscode and IntelliJ to easily apply formatting. In this case, you should add a space after `if` and another space before the `{`

Reply:
Thanks for pointing it out! I'm using ESlint on VS Code, but it's probably configured differently from yours.
Can you update your formatter configuration file to the root folder, for all of us to use the same?